-------------------------------------------------------------------------------
Test set: TestSuite
-------------------------------------------------------------------------------
Tests run: 2, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.51 sec <<< FAILURE! - in TestSuite
Dummy_test_to_fail(test.test)  Time elapsed: 0.021 sec  <<< FAILURE!
java.lang.AssertionError: expected [1] but found [2]
	at test.test.Dummy_test_to_fail(test.java:15)

